utils/kata-manager.sh

```diff
@@ -239,7 +239,7 @@ Description: Install $kata_project [1] (and optionally $containerd_project [2])
 
 Options:
 
- -c <version> : Specify containerd version.
+ -c <flavour> : Specify containerd flavour ("lts" | "active" - default: "lts")
```

Comment:
I'd consider defining "lts". Something like:

```suggestion
 -c <flavour> : Specify containerd flavour ("lts" | "active"
                Defaults to "lts" for the Long Term Support (LTS) version which is appropriate for most users.
```

Reply:
Oh, I've updated the patch while you're reviewing it, and decided to point users to: https://containerd.io/releases/#support-horizon

Is that fine by you?